python-embedder/script.py

```
@@ -0,0 +1,23 @@
+    text = request.json['text']
+
+    # Generate embedding
+    embedding = model.encode(text)
+
+    return jsonify({
+        'embedding': embedding.tolist(),
+        'dimensions': len(embedding)
+    })
+
+if __name__ == '__main__':
+    app.run(host='0.0.0.0', port=8000)
```

Ruff (0.14.4)

Possible binding to all interfaces

(S104)